src/arrayMethodSort.js

```diff
@@ -5,7 +5,30 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    for (let i = 0; i < this.length; i++) {
+      let current = this[i];
+
+      for (let j = i + 1; j < this.length; j++) {
```

Comment:
use `do while`, so you wouldn't iterate through all the array if it is already sorted